pkg/operations/start.go

```diff
@@ -7,21 +7,22 @@ import (
 	"log"
 	"path"
 	"path/filepath"
-	"strings"
 	"time"
 
+	"github.com/weaveworks/ignite/pkg/providers"
+
```

Comment:
remove two blank lines

Reply:
Will do a `make tidy` after the rebase that will get rid of the blank lines.

Reply:
Done in 6478c65.